setup/cli.php

```diff
@@ -43,6 +57,12 @@ function build_container_for_setup() {
 		);
 	};
 
+	$c["agent.global_screen"] = function($c) {
```

Comment:
@klees on the long term, wouldn't be great to have some kind of `Collector` which parses the code-base to collect all Agents?

Reply:
Yes., absolutely.

Luckily you already contributed that `ImplementationOfInterfaceFinder`. I'm just a little reluctant to do this just now, I want to get some other stuff right before I pull in complete ILIAS =)